x/subspaces/keeper/msg_server_test.go

```diff
@@ -554,7 +554,7 @@ func (suite *KeeperTestsuite) TestMsgServer_CreateUserGroup() {
 			shouldErr: true,
 		},
 		{
```

Comment:
Add a test case to check that an error occures if we receive an invalid permission value